cstore_reader.c

```diff
@@ -114,6 +120,12 @@ CStoreBeginRead(const char *filename, TupleDesc tupleDescriptor,
 											  ALLOCSET_DEFAULT_MINSIZE,
 											  ALLOCSET_DEFAULT_INITSIZE,
 											  ALLOCSET_DEFAULT_MAXSIZE);
+	columnCount = tupleDescriptor->natts;
+	projectedColumnMask = ProjectedColumnMask(columnCount, projectedColumnList);
+
```

Comment:
```columnCount``` and ```projectedColumnMask``` are related to ```blockDataBuffersArray``` but not ```stripeReadContext```. So, can we move this new line to the line before ```columnCount```?